actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -196,108 +197,125 @@ pub fn call_generic<RT: Runtime>(
             &[]
         };
 
-        if precompiles::Precompiles::<RT>::is_precompile(&dst) {
-            let context = PrecompileContext {
-                is_static: matches!(kind, CallKind::StaticCall) || system.readonly,
-                gas,
-                value,
-            };
+        'early_ret: {
```

Comment:
what is this apostrophe thing?

Reply:
Is this a block label or something?

Reply:
And what does this any of this have to do with EXT stuff, please keep prs small and focused to what they are supposed to be doing.

If you want to pot puri changes, do them separate.

Reply:
its doing labeled breaks on a block, since the control flow here was getting a bit wonky and IMO the best choice was to short circut (but without doing a closure or making a function there isnt a way to do this sort of thing otherwise). It was stabilized in 1.65 hence the toolchain update (then the new clippy things).

I will split this update into another PR (and to also backport to master)

Reply:
I'm afraid a bit that updating the toolchain now and using new construct in a critical crunch time is a distraction. We should be 100% focused on the absolutely minimum changes.
Did you consider working around this problem with the current toolchain version?

Reply:
Updating the toolchain isn't a huge issue, but I really don't think this feature is worth it.